.bazelci/presubmit.yml

```diff
@@ -565,6 +572,8 @@ tasks:
       - "//..."
     test_targets:
       - "//..."
+    # coverage_targets: # TODO: fix https://github.com/bazelbuild/rules_rust/issues/2838
```

Comment:
Isn't this used for regression testing?

Reply:
correct, but bazel coverage doesn't work on main with this target (even without my changes)

on main, we have 3 broken coverage scenarios whereas with my pr, we'll have 1 broken coverage scenario